### PR TITLE
Move pg operations(commands) from ssh to http

### PR DESCRIPTION
**Pull request by rugwirobaker**
https://github.com/superfly/flyctl/pull/893

### CODE REVIEW

File: pkg/commands/handler.go [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+const Port = 5600
+
+func StartCommandsHandler() {
+
+	http.HandleFunc("/commands/users", handleListUsers)
+	http.HandleFunc("/commands/users/create", handleCreateUser)
+	http.HandleFunc("/commands/users/delete", handleDeleteUser)
+
+	http.HandleFunc("/commands/users/grant", handleGrantAccess)
```
Comment (davissp14) [inline pkg/commands/handler.go:22]: You can ditch the grant/revoke handlers, these operations should be handled by user create/delete.

File: pkg/flycheck/checks.go [hunk 2]
```diff
@@ -12,13 +12,23 @@ import (
...
+	// r := chi.NewRouter()
 
-	fmt.Printf("Listening on port %d", Port)
-	http.ListenAndServe(fmt.Sprintf(":%d", Port), nil)
+	r := http.NewServeMux()
+
+	r.HandleFunc("/flycheck/vm", runVMChecks)
+	r.HandleFunc("/flycheck/pg", runPGChecks)
+	r.HandleFunc("/flycheck/role", runRoleCheck)
+
```
[the remark this answers is not in the document]
Comment (rugwirobaker) [reply pkg/flycheck/checks.go:23]: @davissp14 I've put them on the same port but they are still in different packages that are imported by a new package `pkg/server`. I could rename it though.

File: pkg/commands/handler.go [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+	pg, close, err := getConnection(r.Context())
+	if err != nil {
+		render.Err(w, err)
+		return
+	}
+	defer close()
+
+	name := chi.URLParam(r, "name")
+
+	db, err := admin.FindDatabase(r.Context(), pg, name)
```
Comment (davissp14) [inline pkg/commands/handler.go:107]: This seems to return status code 500 when the the target database doesn't exist.

Flyctl expects 404: https://github.com/superfly/flyctl/blob/be327db0075b8e4bb35eda4c8359da0936b0c42d/pkg/flypg/pg.go#L92

**Review by davissp14 — CHANGES_REQUESTED** [review]
Found a little bug with the attach process.

**Review by davissp14 — APPROVED** [review]
Nice work!

**Pull request merged**